### ISSUE TITLE
Consider adding a `fathom-pick list` command or similar to get the list of sample filenames

### ISSUE THREAD

**Issue by biancadanforth** [mozilla/fathom#97]
[FathomFox](https://github.com/mozilla/fathom-fox)'s Vectorizer can generate a `vectors.json` file for multiple sample pages at once, though it requires the sample pages to be entered as a multiline string where each line is a sample page's filename, e.g.:
```
0001 Walmart.html
0002 Walmart.html
0007 eBay.html
0008 Walmart.html
0010 eBay.html
0011 Walmart.html
0015 amz.html
0016 Walmart.html
0017 Walmart.html
0018 Walmart.html
```

It'd be helpful if there were a command-line tool that output the list of files in a particular directory as a multiline string, so that the list could be copied and pasted for input into the Vectorizer. For example something like `fathom-pick list training` might output such a multiline string for all the files in the `training` directory.

Currently, I have been using `ls > ~/output.txt`, opening this file in a text editor and copying and pasting it from there.

**Comment by erikrose** [comment]
On the Mac, you can just select the files in the Finder, Copy, and Paste into BBEdit to get the list of names. However, I see that other text editors act differently. vi, for example, lists the entire paths. And Daniel wasn't able to find similar functionality readily with Windows Explorer + Notepad. So this could be a nice addition (`fathom-list`?) if https://github.com/mozilla/fathom-fox/issues/31 can't do it more straightforwardly. (In fact, such a util came up during one of my spin-up sessions with Daniel.) Thanks!

**Comment by danielhertenstein** [comment]
Another option is changing how the vecorizer works such that you don't need to input filenames. We could look at the directory specified in the `Base URL:` text box and vectorize all HTML files we see. We would be making an assumption about the user's file structure, but we already expect a certain file structure from using `fathom-pick`.

**Comment by erikrose** [comment]
A couple cautions come to mind:
* Sometimes you want to vectorize only a few files, just to test a hypothesis, especially when experimenting on large corpora.
* By crawling the Python-served index page (which I assume is what you mean), we'd be coding in the assumption that the index is served in the format that Python's `http.server` happens to use. I have a feeling these corpora may someday be on a remote server and served by something else.
* fathom-pick doesn't mandate or even suggest a file structure; it just moves a random selection of files. Perhaps you're thinking of the recommended FS layout from the training docs.

**Comment by danielhertenstein** [comment]
Those are all good cautions. It would be simple enough to make a `fathom-list` tool.

**Comment by danielhertenstein** [comment]
Closed with #100
